web/src/Overview.jsx

```diff
@@ -58,10 +58,41 @@ function Overview() {
   ];
 
   const InstallButton = () => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const open = () => setIsOpen(true);
+    const close = () => setIsOpen(false);
+    const install = () => client.manager.startInstallation();
+
     return (
-      <Button isLarge variant="primary" onClick={() => client.manager.startInstallation()}>
-        Install
-      </Button>
+      <>
+        <Button isLarge variant="primary" onClick={open}>
+          Install
+        </Button>
+
+        <Modal
+          title="Confirm Installation"
+          isOpen={isOpen}
+          showClose={false}
+          variant={ModalVariant.small}
+          actions={[
+            <Button key="accept" variant="primary" onClick={install}>
+              Install
+            </Button>,
+            <Button key="back" variant="primary" onClick={close} autoFocus>
+              Back
```

Comment:
This is the first time we use "Back" in the D-Installer. Usually we use "Cancel". I'd consider rewording the message and change this by _Cancel_, using a "link" variant.